core-bundle/tests/EventListener/DataContainer/DisableBundleConfiguredSettingsListenerTest.php

```diff
@@ -17,16 +17,13 @@
 use Contao\CoreBundle\Tests\TestCase;
 use Contao\Image;
 use Doctrine\Common\Annotations\AnnotationReader;
-use PHPUnit\Framework\MockObject\MockObject;
 use Symfony\Contracts\Translation\TranslatorInterface;
-use Terminal42\ServiceAnnotationBundle\ServiceAnnotationInterface;
 
 class DisableBundleConfiguredSettingsListenerTest extends TestCase
 {
     public function testAnnotatedCallbacks(): void
     {
         $listener = $this->createListener();
-        $this->assertInstanceOf(ServiceAnnotationInterface::class, $listener);
 
         $annotationReader = new AnnotationReader();
         $annotation = $annotationReader->getClassAnnotation(new \ReflectionClass($listener), Callback::class);
```

Comment:
If You won't readd `$this->assertInstanceOf(ServiceAnnotationInterface::class, $listener);` there's no need for an instance of the listener. Then the class string `DisableBundleConfiguredSettingsListener::class` can be used for the reflection.